docs/faq.md

```diff
@@ -0,0 +1,28 @@
+# Versions of the Readme Typing SVG
+
+Image version:
+
+```html
+[![Typing SVG](https://readme-typing-svg.herokuapp.com/?lines=First+line+of+text;Second+line+of+text)](https://git.io/typing-svg)
+```
+
+HTML version:
+
+```html
+  <img src="https://readme-typing-svg.herokuapp.com/?lines=Type+messages+everywhere!;Add+a+bio+to+your+profile!;Add+a+description+to+your+repo!;Make+your+readme+stand+out!&font=Fira%20Code&center=true&width=380&height=50">
```

Comment:
While the link is not required for it to function, it would make it equivalent to the markdown example

```suggestion
<a href="https://git.io/typing-svg"><img src="https://readme-typing-svg.herokuapp.com/?lines=First+line+of+text;Second+line+of+text"></a>
```